docs/source/explore/get-started/vr-controls.rst

```diff
@@ -15,15 +15,114 @@ Use these controls on your VR controllers so that you can enjoy your experience
+
+
++----------------------------+---------------------------------------------------------------------------------+
+| Setting                    | Description                                                                     |
++============================+=================================================================================+
+| Teleporting                | Enable to teleport to a location within a domain.                               |
++----------------------------+---------------------------------------------------------------------------------+
+| Walking                    | Enable this setting to walk in a domain.                                        |
++----------------------------+---------------------------------------------------------------------------------+
+| Strafing                   | Enable for your avatar to move from side to side.                               |
++----------------------------+---------------------------------------------------------------------------------+
+| Jumping and flying         | Enable for your avatar to jump and fly in High Fidelity.                        |
++----------------------------+---------------------------------------------------------------------------------+
+| *Movement Direction:*      | When selected, your avatar will move in the direction your head is facing.      |
+| HMD-Relative               |                                                                                 |
++----------------------------+---------------------------------------------------------------------------------+
+| *Movement Direction:*      | When selected, your avatar will move in the direction your hand is pointing.    |
+| Hand-Relative              |                                                                                 |
++----------------------------+---------------------------------------------------------------------------------+
+| *Movement Direction:*      | When selected, your avatar will move in the direction your hand is pointing,    |
+| Hand-Relative (Level)      | without taking pitch into account.                                              |
++----------------------------+---------------------------------------------------------------------------------+
+| Dominant Hand              | Select 'Left' or 'Right'. Teleport and turning controls move to the controller  |
+|                            | in the dominant hand.                                                           |
++----------------------------+---------------------------------------------------------------------------------+
+| *Control Scheme Selection:*| In Default mode, your walking speed doesn't change depending on how far forward |
+| Default                    | you push your controller's joystick. Fully pushing your joystick forward will   |
+|                            | make your avatar run at twice your walking speed.                               |
```

Comment:
```suggestion
|                            | make your avatar run.                               |
```

I don't think its important to mention that running is twice your walking speed. I think most people think of run as "faster than walking" so its needless to say what the technical detail of that is. Plus, once you get into Analog and Analog++, what does twice the walking speed mean since your walkign speed varies based on how far forward you push the joystick. So we'll stick with "walk or run".